client/my-sites/upgrades/checkout-thank-you/google-voucher/index.jsx

```diff
@@ -115,7 +115,7 @@ class GoogleVoucherDetails extends Component {
 					text={ this.props.translate( 'Generate code' ) } />
 
 				<TipInfo
-					info={ this.props.translate( 'Offer valid in US and Canada after spending the first $25 on Google AdWords.' ) } />
+					info={ this.props.translate( 'Offer valid in US after spending the first $25 on Google AdWords.' ) } />
```

Comment:
Warning: 17 translations will be lost with this change.
Alternate string suggestion: 
* `translate( '$100 Google AdWords credit after spending the first $25. Offer valid in US and Canada.' )`&mdash; translations: **6**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/update-google-adwords-texts)